ff2mpv.py

```diff
@@ -47,6 +96,22 @@ def get_message():
     return json.loads(message)
 
 
+def get_whitelist():
+    path = get_config_path('whitelist.txt')
+    if path == '':
+        return ['a^'] # impossible regex
+    file = open(path, 'r')
+    return [line[:-1] for line in file.readlines()] # strip newline characters
+
+
+def is_whitelisted(url, whitelist):
```

Comment:
You could make this as simple as an `any` comprehension:

```python
def is_whitelisted(url, whitelist):
    return any(pattern.match(url) for pattern in whitelist)
```

Reply:
Done